doc/en/cache.rst

```diff
@@ -262,16 +256,96 @@ You can always peek at the content of the cache using the
+       'failure_demo.py::TestSpecialisedExplanations::test_eq_similar_text': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_eq_text': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_in_list': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_not_in_text_multiline': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_not_in_text_single': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_not_in_text_single_long': True,
+       'failure_demo.py::TestSpecialisedExplanations::test_not_in_text_single_long_term': True,
+       'failure_demo.py::test_attribute': True,
+       'failure_demo.py::test_attribute_failure': True,
+       'failure_demo.py::test_attribute_instance': True,
+       'failure_demo.py::test_attribute_multiple': True,
+       'failure_demo.py::test_dynamic_compile_shows_nicely': True,
+       'failure_demo.py::test_generative[3-6]': True,
+       'test_50.py::test_num[17]': True,
        'test_50.py::test_num[25]': True,
+       'test_anothersmtp.py::test_showhelo': True,
        'test_assert1.py::test_function': True,
        'test_assert2.py::test_set_comparison': True,
+       'test_backends.py::test_db_initialized[d2]': True,
        'test_caching.py::test_function': True,
-       'test_foocompare.py::test_compare': True}
+       'test_checkconfig.py::test_something': True,
+       'test_class.py::TestClass::test_two': True,
+       'test_compute.py::test_compute[4]': True,
+       'test_example.py::test_error': True,
+       'test_example.py::test_fail': True,
+       'test_foocompare.py::test_compare': True,
+       'test_module.py::test_call_fails': True,
+       'test_module.py::test_ehlo': True,
+       'test_module.py::test_ehlo[mail.python.org]': True,
+       'test_module.py::test_ehlo[smtp.gmail.com]': True,
+       'test_module.py::test_event_simple': True,
+       'test_module.py::test_fail1': True,
+       'test_module.py::test_fail2': True,
+       'test_module.py::test_func2': True,
+       'test_module.py::test_interface_complex': True,
+       'test_module.py::test_interface_simple': True,
+       'test_module.py::test_noop': True,
+       'test_module.py::test_noop[mail.python.org]': True,
+       'test_module.py::test_noop[smtp.gmail.com]': True,
+       'test_module.py::test_setup_fails': True,
+       'test_parametrize.py::TestClass::test_equals[1-2]': True,
+       'test_sample.py::test_answer': True,
+       'test_show_warnings.py::test_one': True,
+       'test_simple.yml::hello': True,
+       'test_smtpsimple.py::test_ehlo': True,
+       'test_step.py::TestUserHandling::test_modification': True,
+       'test_strings.py::test_valid_string[!]': True,
+       'test_tmp_path.py::test_create_file': True,
+       'test_tmpdir.py::test_create_file': True,
+       'test_tmpdir.py::test_needsfiles': True,
+       'test_unittest_db.py::MyTest::test_method1': True,
+       'test_unittest_db.py::MyTest::test_method2': True}
```

Comment:
This looks unexpected?

Reply:
Hmmm you are right, I missed that.

Probably it is picking up the cache from the previous runs; I get the same output locally even if I remove my `.pytest_cache` folder. Not sure why this is happening just now.

Regardless we can merge this and fix it on `master`, I think.

Reply:
Why not fix it on master before merging?

Reply:
(it is not clear to me / I don't know how that works though - I just think the less the diffs the better)

Reply:
Yeah, it is also in https://github.com/pytest-dev/pytest/pull/5014/files#diff-040438ee0e8d77242351c8cc6baf8499L269 already - gonna fix it from there.

Reply:
I think it should even hold the release back, since it would have wrong documentation.

Reply:
But that is tagged already, so ok.

Reply:
But I still think we should not merge regressions across branches.

Reply:
No worries, I will close this for now and we can merge this later when this issue is fixed. 👍

Reply:
Any idea about the issue?
Would be nice to have this merged for the version at least.. :)

Reply:
Haven't had the time to look at it yet. 😞 

> for the version at least

What do you mean? 🤔

Reply:
`--version` - to get the tag into features.  Just something minor.